docs/package/gluon-ssid-changer.md

```diff
@@ -0,0 +1,70 @@
+gluon-ssid-changer
+==================
+
+This package adds a script to change the SSID to an Offline-SSID when there is
```

Comment:
not really, because it only changes from online to offline every timeframe, but could change back from offline to online every minute.

but i will rephraze this to be clearer.